tests/testthat/test-get_source_expressions.R

```diff
@@ -249,6 +249,13 @@ test_that("#1262: xml_parsed_content gets returned as missing even if there's no
   expect_identical(source_expressions$expressions[[1L]]$full_xml_parsed_content, xml2::xml_missing())
 })
 
```

Comment:
nit: i usually like to add tests from duplicate issues just to ensure future updates don't regress one issue but not the others, i.e. just copy-paste the reported reprex with expected output. i think we're pretty confident that the issues are the exact same root cause, but better safe than sorry.